app/helpers/books_helper.rb

```diff
@@ -0,0 +1,2 @@
+module BooksHelper
```

Comment:
Created by rails while generating controller. Should I remove this from project if not used?